src/main/java/picard/sam/BamErrorMetric/CollectBamErrorMetrics.java

```diff
@@ -0,0 +1,464 @@
+)
+public class CollectBamErrorMetrics extends CommandLineProgram {
+    @Argument(shortName = StandardOptionDefinitions.INPUT_SHORT_NAME, doc = "Input SAM or BAM file.")
+    public File INPUT;
+
+    @Argument(shortName = StandardOptionDefinitions.OUTPUT_SHORT_NAME, doc = "Basename for output files.")
+    public File OUTPUT;
+
+    @Argument(doc = "Errors to collect in the form of \"ERROR(,STRATIFIER)*\". " +
+            "To see the values available for ERROR and STRATIFIER look at the documentation for the arguments " +
+            "ERROR_VALUE and STRATIFIER_VALUE.")
+    public List<String> ERROR_METRICS = new ArrayList<>(CollectionUtil.makeList(
+            "ERROR",
+            "ERROR,BASE_QUALITY",
+            "ERROR,INSERT_LENGTH",
+            "ERROR,GC_CONTENT",
+            "ERROR,READ_DIRECTION",
+            "ERROR,PAIR_ORIENTATION",
+            "ERROR,HOMOPOLYMER",
+            "ERROR,BINNED_HOMOPOLYMER",
+            "ERROR,CYCLE",
+            "ERROR,READ_ORDINALITY",
+            "ERROR,READ_ORDINALITY,CYCLE",
+            "ERROR,READ_ORDINALITY,HOMOPOLYMER",
+            "ERROR,READ_ORDINALITY,GC_CONTENT",
+            "ERROR,READ_ORDINALITY,PRE_DINUC",
+            "ERROR,MAPPING_QUALITY",
+            "ERROR,READ_GROUP",
+            "ERROR,MISMATCHES_IN_READ",
+            "ERROR,ONE_BASE_PADDED_CONTEXT",
+            "OVERLAPPING_ERROR",
+            "OVERLAPPING_ERROR,BASE_QUALITY",
+            "OVERLAPPING_ERROR,INSERT_LENGTH",
+            "OVERLAPPING_ERROR,READ_ORDINALITY",
+            "OVERLAPPING_ERROR,READ_ORDINALITY,CYCLE",
+            "OVERLAPPING_ERROR,READ_ORDINALITY,HOMOPOLYMER",
+            "OVERLAPPING_ERROR,READ_ORDINALITY,GC_CONTENT"));
+
+    @Argument(doc = "A fake argument used to show what the options of ERROR (in ERROR_METRICS) are.", optional = true)
+    public BaseErrorCalculation.Errors ERRORS_VALUE = null;
+
+    @Argument(doc = "A fake argument used to show what the options of STRATIFIER (in ERROR_METRICS) are.", optional = true)
+    public ReadBaseStratification.Stratifiers STRATIFIER_VALUE = null;
+
+    @Argument(shortName = "V", doc = "VCF of loci to avoid collecting data (should include any known variation!)")
+    public File VCF=null;
+
+    @Argument(shortName = "L", doc = "Region(s) to limit analysis to. Can be given as a interval_list file. (will intersect if multiple values are given.) ", optional = true)
+    public List<File> INTERVALS = null;
+
+    @Argument(shortName = StandardOptionDefinitions.MINIMUM_MAPPING_QUALITY_SHORT_NAME, doc = "Minimum Mapping quality to include read.", minValue = 0)
+    public int MIN_MAPPING_Q = 20;
+
+    @Argument(shortName = "BQ", doc = "Minimum Base quality to include base.",minValue = 0)
+    public int MIN_BASE_Q = 20;
+
+    @Argument(shortName = "PE", doc = "The prior error, in phred-scale (used for calculating ratios and filtering out HET sites)",
+            optional = true, minValue = 2)
+    public int PRIOR_Q = 30;
```

Comment:
Without reading the code, I have no idea what the prior is used for.  The description the brackets don't help.  I tried reading the code and couldn't figure it out either :/